### DISCUSSION TITLE
Task list: Set assignees from mentions in list items

### DISCUSSION THREAD

**Discussion by bennycode** [community/community#4465] [Projects and Issues]
To know who is supposed to work on a specific item, my team and I are using mentions behind list item entries:

**Example:**

![image](https://user-images.githubusercontent.com/469989/124280468-fe915e00-db48-11eb-9b50-d52669b77d8d.png)

When clicking on "Convert to issue" it would be nice if the person from the mention automatically becomes assigned to the newly created issue. That would speed up the issue creation flow for us.

**Comment by ckarenz** [comment]
Related to https://github.com/github/feedback/discussions/4464.

My team also uses @ mentions to informally assign work.  Right now we're assigning separately in the task and the task list so we can see at a glance who's on the hook.  This is onerous and pushing us back to project boards.

It would be great if the assignee was included in the expanded text.  This could be bidirectional (assign from mention and show mentions on change to assignment), but really either way fills the need.
